dex/testing/eth/harness.sh

```diff
@@ -202,8 +203,6 @@ do
   "${NODES_ROOT}/harness-ctl/mine-alpha" "5"
 done
 
-# Transactions can take an eternity to be mined...
-# TODO: Determine why this is.
```

Comment:
In a nutshell, why was this?

Reply:
I think it comes down to a race here https://github.com/ethereum/go-ethereum/blob/f05419f0fb8c5328dca92ea9fb184d082300344a/miner/worker.go#L387-L409

If you stop before this switch can make it down to including more transactions at `<-timer.C`, it just doesn't include them.

Reply:
Hmm, looking at the git blames, I see they made the timer duration adjustable: https://github.com/ethereum/go-ethereum/pull/17444

Reply:
Hadn't noticed the `RecommitInterval` setting until now.

Reply:
I tried some logs, and it does seem that it's the case that txs aren't included when we stop before the timer can check there, I am not sure where the txs are sometimes included even when we don't make it there, however.